jobs/storage_test.groovy

```diff
@@ -0,0 +1,57 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = "storage_backend"
```

Comment:
this job currently hardcodes `gcs` as the storage backend... maybe we rename this job to `gcs_storage_backend_e2e` -- unless there are plans for this one job to test multiple storage backends...